assets/javascripts/discourse/templates/components/realtime-validation-settings.hbs

```diff
@@ -0,0 +1,32 @@
+<div>
+    <label>{{i18n 'admin.wizard.field.validations.header'}}</label>
+</div>
+{{#each-in field.validations as |name props|}}
+  <span class="setting-title">
+    {{input type="checkbox" checked=props.status }}
```

Comment:
@fzngagan As well as this being a combo-box, the property itself should probably be called ``type`` rather than ``status``, as we're talking about the type of validation here.

Reply:
rename status to enabled